src/main/java/ru/kovalchuk/Impl/ConsolePrinter.java

```diff
@@ -0,0 +1,23 @@
+package ru.kovalchuk.Impl;
+
+import lombok.extern.slf4j.Slf4j;
+import ru.kovalchuk.Printer;
+import ru.kovalchuk.Task;
+
+import java.util.List;
+
+@Slf4j
+public class ConsolePrinter implements Printer {
+    private static final String inProgress = ". [ ] "; // задача не выполнена
+    private static final String done = ". [v] "; // задача выполнена
+
+    @Override
+    public void printTasks(List<Task> listTask) {
+        listTask.forEach(task -> log.info(representTask(task)));
```

Comment:
Я бы сделал здесь вот таким образом, чтобы дважды не гулять по одной и той же коллекции:
```suggestion
        listTask
            .stream()
            .map(ConsolePrinter::representTask)
            .forEach(taskString -> { 
                log.info(taskString); 
                System.out.println(taskString)
            });
```

Reply:
Исправлено